composer.circleci.json

```diff
@@ -34,7 +34,7 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "scripts": {
-        "test": "./vendor/bin/phpunit --configuration phpunit.circleci.xml"
+        "test": "./vendor/bin/phpunit
```

Comment:
Argh... just noticed the typo I made here (missing end-`"`). Sorry about that. EVen so, as we'll be moving away from Circle, this will be fixed in the next PR when I remove this file.